BraveShared/tr.lproj/Localizable.strings

```diff
@@ -0,0 +1,1226 @@
+/* The title of the screen explaining Brave Shields */
+"AboutBraveShields" = "Brave Kalkanları Hakkında";
+
+/* The body of the screen explaining Brave Shields */
+"AboutBraveShieldsBody" = "Siteler genellikle sizi ve cihazınızı tanımlamaya çalışan çerezler ve komut dosyaları içerir. Kim olduğunuzu öğrenmek ve sizi web'de takip etmek ve her sitede ne yaptığınızı izlemek istiyorlar.\n\nBrave, takip edilmeden gezinebilmeniz için bunları engeller.";
+
+/* No comment provided by engineer. */
+"BraveRewardsAdNotificationTitle" = "Brave Ödülleri";
+
+/* No comment provided by engineer. */
+"BraveRewardsOpen" = "Açık";
+
+/* No comment provided by engineer. */
+"BraveShieldsAdvancedControls" = "Gelişmiş denetimler";
+
+/* The number of ads and trackers blocked will be next to this */
+"BraveShieldsBlockedCountLabel" = "Reklamlar ve diğer ürpertici şeyler engellendi";
+
+/* What the screen reader will read out when the user has VoiceOver on and taps on the question-mark info button on the shields panel */
+"BraveShieldsBlockedInfoButtonAccessibilityLabel" = "Daha fazla bilgi edinin";
+
+/* No comment provided by engineer. */
+"BraveShieldsGlobalChangeButton" = "Genel Kalkan varsayılanlarını değiştirme";
+
+/* No comment provided by engineer. */
+"BraveShieldsGlobalControls" = "Genel Denetimler";
+
+/* No comment provided by engineer. */
+"BraveShieldsSiteBroken" = "Bu site bozuk görünüyorsa, Kalkanları indirerek deneyin";
+
+/* Context: 'Brave Shields Up' or 'Brave Shields Down' */
+"BraveShieldsStatusTitle" = "Brave Kalkanları";
+
+/* Context: The 'Down' in 'Brave Shields Down' */
+"BraveShieldsStatusValueDown" = "Aşağı";
+
+/* Context: The 'Up' in 'Brave Shields Up' */
+"BraveShieldsStatusValueUp" = "Yukarı";
+
+/* The accessibility hint spoken when focused on the main shields toggle */
+"BraveShieldsToggleHint" = "Brave Kalkanları'nı değiştirmek için iki kez dokunun";
+
+/* Button title for Adding Engine in navigation Bar */
+"customSearchEngine.addButtonTitle" = "eklemek";
+
+/* Label explaining how to add search engine. */
+"customSearchEngine.addEngineDescription" = "Arama URL'sini yazın ve sorguyu %1$s ile değiştirin.\nÖrneğin: https://youtube.com/search?q= %2$s \n(Site OpenSearch'i destekliyorsa, bu alan düzenlenirken otomatik olarak ekleme seçeneği sağlanır.)";
```

Comment:
`"customSearchEngine.addEngineDescription" = "Arama URL'sini yazın ve sorguyu %s ile değiştirin.\nÖrneğin: https://youtube.com/search?q= %s \n(Site OpenSearch'i destekliyorsa, bu alan düzenlenirken otomatik olarak ekleme seçeneği sağlanır.)";`